src/features/content-answers/ContentAnswersModalError.scss

```diff
@@ -0,0 +1,29 @@
+@import '../../styles/variables';
+
+.bdl-ContentAnswersModalError {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+    width: 100%;
```

Comment:
is this necessary?

Reply:
yea, without it, it wasnt centering in the modal, content within the modal isnt center by default

Reply:
the `width: 100%`?

Reply:
oh you meant the width? yea we can remove that

Reply:
removed